mmhuman3d/data/datasets/human_image_dataset.py

```diff
@@ -109,6 +109,10 @@ def prepare_raw_data(self, idx: int):
         image_path = self.human_data['image_path'][idx]
         info['image_path'] = os.path.join(self.data_prefix, 'datasets',
                                           self.dataset_name, image_path)
+        if image_path.endswith('smc'):
```

Comment:
We may add a test for this case in [tests/test_datasets/test_human_image_dataset.py](https://github.com/open-mmlab/mmhuman3d/blob/main/tests/test_datasets/test_human_image_dataset.py).

Reply:
Added in e67820f98c0353290bd899b3a29516b30d44d7b6.